config/sources/families/k3.conf

```diff
@@ -20,13 +20,28 @@ declare -g SPD_OPTEED="SPD=opteed"
 
 declare -g INSTALL_HEADERS="yes"
 
-declare -g TI_PACKAGES=()
+declare -g TI_PACKAGES=("k3conf")
 if [[ "${CC33XX_SUPPORT}" == "yes" ]] ; then
 	if [[ "${RELEASE}" == "trixie" || "${RELEASE}" == "noble" ]] ; then
 		TI_PACKAGES+=("cc33xx-fw" "cc33xx-target-scripts" "cc33conf" "cc33calibrator")
 	fi
 fi
 
+if [[ "${GPU_SUPPORT}" == "yes" ]] ; then
+	TI_PACKAGES+=("ti-img-rogue-driver-${SOC_ID}-dkms" "ti-img-rogue-umlibs-${SOC_ID}" "ti-img-rogue-tools-${SOC_ID}" "ti-img-rogue-firmware-${SOC_ID}")
+	TI_PACKAGES+=("mesa-vulkan-drivers" "libgl1-mesa-dri")
+fi
+
+if [[ "${HAS_VIDEO_OUTPUT}" != "no" ]] ; then
+	TI_PACKAGES+=("weston" "emptty")
+fi
+
+TI_PACKAGES+=("firmware-cnm-wave" "firmware-ti-ipc-${SOC_ID}" "firmware-ti-connectivity")
```

Comment:
We should fix up the packaging to that `firmware-ti-ipc` is not SoC specific. The files are small and do not conflict with each other, better to just have a common package, that is more inline with the "Debian" way of packaging.

Reply:
good point!

if you want to push some files to a package armbian-ti-common in APA, just let me know.